controllers/v1/assessments.js

```diff
@@ -49,15 +49,23 @@ exports.listPrograms = async function (req, res) {
         res.send(response);
     }
     else {
+
         //get quey from cassandra
         model.MyModel.findOneAsync({ qid: "list_assessment_programs_query" }, { allow_filtering: true })
             .then(async function (result) {
-                var bodyParam = JSON.parse(result.query);
+
+                let bodyParam = JSON.parse(result.query);
+
                 if (config.druid.assessment_datasource_name) {
                     bodyParam.dataSource = config.druid.assessment_datasource_name;
                 }
-                bodyParam.filter.dimension = req.body.entityType;
-                bodyParam.filter.value = req.body.entityId;
+               
+                let userId = await getCreatedByField(req, res);
```

Comment:
@deepakr123 - Please evaluate if we can make createdBy as an array.

Reply:
resolved